Exercícios/Para casa/entregas/MARIANA_INOUE/SavingAccount.js

```diff
@@ -0,0 +1,67 @@
+const { BankAccount } = require('./BankAccount')
+const { Client } = require('./Client');
+const { Bank } = require('./Bank');
+
+class SavingAccount extends BankAccount {
+    incomeDay;
+    incomeRate;
+    maxFreeWithdrawals = 3;
+    #qtdWithdrawal;
+    #withdrawalTax
+    constructor(client, bank, accountNumber, agencyNumber, incomeDay, incomeRate) {
+
+        super(client, bank, accountNumber, agencyNumber)
+        this.incomeDay = incomeDay;
+        this.incomeRate = incomeRate;
+        this.#withdrawalTax = 0.03
+        this.#qtdWithdrawal = 0
+
+    }
+
+    get qtdWithdrawal() {
+        return this.#qtdWithdrawal
+    }
+
+    get withdrawalTax() {
+        return this.#withdrawalTax
+    }
+    set withdrawalTax(newWithdrawalTax) {
+        this.#withdrawalTax = newWithdrawalTax
+    }
+
+    generateIncome(currentDay) {
+        if (this.incomeDay === currentDay) {
+            this.balanceAdd(this.incomeRate)
+            console.log(`Hoje é dia ${this.incomeDay} e taxa rendimento será de ${this.incomeRate}.`)
+        } else {
+            console.log(`Sem rendimento para hoje.`)
+        }
+    }
+
+    cashWithdrawal(amount) {
+        console.log(`Informação, conta poupança: Até 2 saques não haverá cobrança de taxas.`)
```

Comment:
No console, ao invés de escrever a quantidade de saques, poderia utilizar `${this.maxFreeWithdrawals}`.